modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -6,7 +6,7 @@
 [id="modifying-retention-time-and-size-for-prometheus-metrics-data_{context}"]
 = Modifying the retention time and size for Prometheus metrics data
 
-By default, Prometheus automatically retains metrics data for 15 days. You can modify the retention time for
+By default, Prometheus automatically retains metrics data for 11 days. You can modify the retention time for
```

Comment:
for prometheus, default retention is 15 days, since we configured ROSA cluster for monitoring with settings in https://github.com/openshift/managed-cluster-config/blob/master/resources/cluster-monitoring-config/config.yaml, and the retention time is changed to 11 days, then
`By default, Prometheus automatically retains metrics data for 11 days` is fine.

another change should be updated
https://66138--docspreview.netlify.app/openshift-rosa/latest/monitoring/configuring-the-monitoring-stack#modifying-retention-time-and-size-for-prometheus-metrics-data_configuring-the-monitoring-stack

update
```
If you do not explicitly define values for either retention or retentionSize, retention time defaults to 15 days, and retention size is not set.
```
to
```
If you do not explicitly define values for either retention or retentionSize, retention time defaults to 11 days, and retention size is not set.
```

Reply:
Changes made. Thanks @juzhao!